<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinhaJanela</class>
 <widget class="QMainWindow" name="MinhaJanela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MinhaJanela</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblPlaca">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Placa:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdicionar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar</string>
    </property>
    <property name="icon">
     <iconset resource="content.qrc">
      <normaloff>:/icones/adic.png</normaloff>:/icones/adic.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
    <property name="icon">
     <iconset resource="content.qrc">
      <normaloff>:/icones/file_edit.png</normaloff>:/icones/file_edit.png</iconset>
    </property>
   </widget>
   <widget class="QTreeWidget" name="twListagem">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>531</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Placa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Marca</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Modelo</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="txtPlaca">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Placa...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtModelo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Modelo...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblModelo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Modelo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Cor...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMarca">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Marca...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMarca">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>52</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Marca:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemover">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Remover</string>
    </property>
    <property name="icon">
     <iconset resource="content.qrc">
      <normaloff>:/icones/remove-01-128.png</normaloff>:/icones/remove-01-128.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOp_es">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuOp_es"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="content.qrc"/>
 </resources>
 <connections/>
</ui>
